distributed/cli/dask_worker.py

```diff
@@ -339,13 +347,20 @@ def del_pid_file():
 
     loop = IOLoop.current()
 
+    if worker_class is not None:
+        worker_class = import_term(worker_class)
+        if nanny:
+            kwargs["worker_class"] = worker_class
+    else:
+        worker_class = Worker
```

Comment:
Maybe change the default to `"dask.distributed.Worker"` and avoid the `is not None` case?

Reply:
Done. Thanks for the suggestion, that's cleaner